cpp/log/log_verifier.h

```diff
@@ -33,7 +33,7 @@ class LogVerifier {
     INVALID_MERKLE_PATH,
```

Comment:
Yes I meant adding a detail message to Status where appropriate, for
logging and potentially error responses to clients.

On 10 November 2015 at 13:20, Pierre Phaneuf <notifications@github.com>
wrote:

> In cpp/log/log_verifier.h
> <https://github.com/google/certificate-transparency/pull/1038#discussion_r44404708>
> :
>
> > @@ -33,7 +33,7 @@ class LogVerifier {
> >      INVALID_MERKLE_PATH,
>
> You mean, in addition to the normal util::Status code and message?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/certificate-transparency/pull/1038/files#r44404708>
> .
>